integration/test_qca.in

```diff
@@ -0,0 +1,6 @@
+(FIND-QCA-PATH : TARGET ("SHC1") : SOURCE ("IRS1") : RELTYPE ("controls-state-change-of"))
```

Comment:
A few other questions about the message format: how does the QCA respond if no path is found? Presumably ```(SUCCESS :paths ())```? Is the response the same of either of the nodes is not found in the network(s) at all?